projects/m1/018-roulette-payouts/solutions/bisicchiamatteo/018-roulette-payouts.py

```diff
@@ -0,0 +1,39 @@
+import random
+
+
+ROULET=["0","1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16", "18", "19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","00"]
+RED=["1","3","5" , "7", "9", "12", "14", "16", "18", "19", "21", "23", "25", "27", "30", "32", "34" , "36"]
```

Comment:
In questo caso c'è una logica ben precisa che ti evita di imputare a mano i valori nell'array.

I numeri minori di 10 sono dispari
da 11 a 19 sono pari
da 20 a 29 sono dispari
da 30 a 39 sono pari